cpp/tests/strings/split_tests.cpp

```diff
@@ -339,6 +305,127 @@ TEST_F(StringsSplitTest, SplitRecordWhitespaceWithMaxSplit)
   CUDF_TEST_EXPECT_COLUMNS_EQUAL(result->view(), expected);
 }
 
+TEST_F(StringsSplitTest, SplitRegex)
```

Comment:
Tests look great overall. Should we consider tests involving multi-byte UTF-8 characters? Are those supported?

Reply:
The `é` is a multi-byte UTF-8 character. I use it in the data and in the regex pattern.

Reply:
Thanks for the correction. I had forgotten that é was multi-byte. I always think of more obscure cases like emoji. 👍